src/python/botan3.py

```diff
@@ -2138,3 +2149,21 @@ def zfec_decode(k, n, indexes, inputs):
         c_size_t(k), c_size_t(n), c_indexes, c_inputs, c_size_t(share_size), c_outputs
     )
     return [output.raw for output in outputs]
+
+#
+# TPM2
+#
+class TPM2Context:
+    def __init__(self, tcti_nameconf: str = None) -> None:
+        tcti = c_char_p(0 if not tcti_nameconf else tcti_nameconf.encode("utf-8"))
+
+        self.__obj = c_void_p(0)
+        rc = _DLL.botan_tpm2_ctx_init(byref(self.__obj), tcti)
+        if rc == -40: # 'Not Implemented'
```

Comment:
Any chance we could pull this magic number from the enum? 😅 
(there are other places that would benefit from it)

Reply:
Mhh, there seems to be some secret sauce around the translation of parameter/result types from C to Python and back. This blog post provides a little inspiration: https://v4.chriskrycho.com/2015/ctypes-structures-and-dll-exports.html

Perhaps we could actually mirror the error-enum into `botan3.py`. That's a manual duplication, but should be okay, given that this enum probably doesn't change all the time. And then we could make CDLL do the translation for us.

Any of this should certainly not go into this pull request, though.